source/config/__init__.py

```diff
@@ -62,10 +62,17 @@ def isInstalledCopy():
 	except WindowsError:
 		return False
 
+
+#: #6864: The name of the subkey stored under NVDA_REGKEY where the value is stored which will make an installed NVDA load the user configuration either from the local or from the roaming application data profile.

```

Comment:
Done, and did some more splits here to at least have a separate comment line per sentence